lib/glific/access_control.ex

```diff
@@ -0,0 +1,285 @@
+defmodule Glific.AccessControl do
+  @moduledoc """
+  The AccessControl context.
+  """
+
+  import Ecto.Query, warn: false
+
+  alias Glific.{
+    AccessControl.FlowRole,
+    AccessControl.Permission,
+    AccessControl.Role,
+    Repo,
+    Users.User
+  }
+
+  @doc """
+  Returns the list of roles.
+
+  ## Examples
+
+      iex> list_roles()
+      [%Role{}, ...]
+  """
+  @spec list_roles(map()) :: [Role.t()]
+  def list_roles(args) do
+    check_roles_and_permission_toggle?(args.organization_id)
```

Comment:
Did not understand this function name. what does it do?

Reply:
@pankaj-ag   I have updated the functions docs now
```
Checks if roles and permission flag is enabled first.
  If roles and permission flag is enabled then list all roles
  If roles and permission flag is disabled then list only default roles
```